webapp/src/components/ParcelDetailPage/ParcelDetail/ParcelMortgage/ParcelMortgage.js

```diff
@@ -27,23 +27,13 @@ export default class ParcelMortgage extends React.PureComponent {
 
   render() {
     const { mortgage } = this.props
+
     return (
-      <Grid className={`ParcelMortgageDetail ${mortgage.status}`}>
+      <Grid className={`ParcelMortgageDetail ${getMortgageStatus(mortgage)}`}>
```

Comment:
Consider caching the `const mortgageStatus = getMortgageStatus(mortgage)` on a variable if it's going to be called > 2 times